src/pfe/file-watcher/server/test/scripts/utils.js

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+if (process.argv.length < 3) {
+    printUsage();
+}
+
+const option = process.argv[2];
+
+if (option.includes("help") || option == "-h") {
+    printUsage();
+} else if (option == "encode") {
+    if (process.argv[3] == undefined || process.argv[4] == undefined || process.argv.length != 5) {
+        printUsage();
+    }
+    encodeCredentials(process.argv[3], process.argv[4]);
+} else if (option == "decode") {
+    if (process.argv[3] == undefined || process.argv.length != 4) {
```

Comment:
same here, just `if (process.argv.length < 4)`

Reply:
updated, Idk why i did that 😅